src/commands/support/remoteEfb.ts

```diff
@@ -0,0 +1,20 @@
+import { MessageCommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed, makeLines } from '../../lib/embed';
+
+const remoteEfbEmbed = makeEmbed({
+    title: 'FlyByWire Support | Remote EFB',
+    description: makeLines([
+        'Currently the EFB cannot be viewed remotely, but it is a planned feature.',
+        '',
+        'In the meantime, you can however use an application like [Spacedesk](https://www.spacedesk.net/)* to create a virtual screen using yout tablet/phone/pc, and then de-attach the EFB from MSFS by holding the ALTGR button and clicking on the EFB with your mouse. You can then drag the EFB window to your virtual screen. Please note that this solution will degrade your performance slightly, as your computer needs to render and process the new virtual screen.',
```

Comment:
```suggestion
        'In the meantime, you can however use an application like [Spacedesk](https://www.spacedesk.net/)* to create a virtual screen using your tablet/phone/pc, and then de-attach the EFB from MSFS by holding the ALTGR button and clicking on the EFB with your mouse. You can then drag the EFB window to your virtual screen. Please note that this solution will degrade your performance slightly, as your computer needs to render and process the new virtual screen.',
```
Small typo correction.